libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -157,10 +159,16 @@ - (void)setConfig:(SFSDKLoginViewControllerConfig *)config {
 #pragma mark - Setup Navigation bar
 
 - (void)setupNavigationBar {
-    self.navBar = [[UINavigationBar alloc] initWithFrame:CGRectZero];
-   
-    // Setup top item.
-    self.navBar.items = @[[self createTitleItem]];
+    NSString *title = [SFSDKResourceUtils localizedString:@"TITLE_LOGIN"];
```

Comment:
We haven't pulled in https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/2405 yet. I'll pull it in and update these code after merging this one. cc: @trooper2013